tests/test_validators.py

```diff
@@ -708,7 +708,7 @@ def test_validator(cls, v, foobar):
 
     with pytest.raises(ConfigError) as exc_info:
         make_generic_validator(test_validator)
-    assert ': (cls, v, foobar), should be: (cls, value, values, config, field)' in str(exc_info.value)
+    assert ': (cls, v, foobar), should be: (cls, value, values, field, config)' in str(exc_info.value)
```

Comment:
Will this break existing code or is it just the error message that is ordered now?

And as a nice feature of pytest, if you use the "matches" argument in `raises` (https://docs.pytest.org/en/stable/assert.html#assertions-about-expected-exceptions) the error message will be nicer when the test fails: 

```suggestion
    expected_str =  r',*: (cls, v, foobar), should be: (cls, value, values, field, config).*'
    with pytest.raises(ConfigError, match=expected_str) as exc_info:
        make_generic_validator(test_validator)
```

Reply:
There was a bug with a validator error message with incorrect order of arguments. Correct order is `(cls, value, values, field, config)` but error message showed `(cls, value, values, config, field)`.

I will update PR to use `match` in `pytest.raises`.

Thanks, for review.